jdi-light-material-ui-tests/src/main/java/io/github/com/MaterialNavigator.java

```diff
@@ -12,6 +12,8 @@
     private static WebDriver driver = getDriver();
     private static String oneSectionLocator = "//div[@class='MuiList-root']//span[text()='%s']";
     private static String twoSectionLocator = "//div[@class='MuiListItemText-root']//span[text()='%s']";
+    private static String sectionTitle = "//h1";
```

Comment:
Please use in title name of the title as it could be different pages opened on by one and for all this pages there is //h1